charts/reform-scan-blob-router/values.preview.template.yaml

```diff
@@ -66,6 +68,11 @@ reformblobstorage:
     containers:
       - bulkscan
       - bulkscan-rejected
+      - cmc
+      - divorce
+      - finrem
+      - probate
+      - sscs
```

Comment:
don't need this for preview ?

Reply:
The way we access blobs we gather containers from configuration and try to access them all. To avoid unnecessary errors or unknown failures - adding all containers from configuration